src/main/Video.tsx

```diff
@@ -134,18 +134,26 @@ const VideoPlayer: React.FC<{url: string, isMuted: boolean}> = ({url, isMuted})
     }
   })
 
+  const playerWrapper = css({
+    backgroundColor: 'rgba(255,255,255,1)',
```

Comment:
Probably either want to leave this out (so it defaults to the globally defined backgroundColor), or pick a color with a higher contrast, maybe black. I'd prefer the global default.

Reply:
changed